components/pom.xml

```diff
@@ -338,6 +338,7 @@
         <module>camel-jooq</module>
         <!-- observability-services requires the availability of other components -->
         <module>camel-observability-services</module>
+        <module>camel-dapr</module>
```

Comment:
move this up so its in the sorted list with the other components that start with a d